src/material-experimental/mdc-list/list-base.ts

```diff
@@ -198,6 +229,18 @@ export abstract class MatInteractiveListBase extends MatListBase
     this._subscriptions.unsubscribe();
   }
 
+  getItems() {
```

Comment:
just make the original symbols public instead of protected, but leave the `_` in the name to indicate that they're internal